### PR TITLE
Remove various todos

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: app/models/track.rb [hunk 1]
```diff
@@ -63,14 +62,20 @@ def top_10_contributors
       to_a
   end
 
-  # TODO: Erik: Implement
   def num_code_contributors
-    10
+    User::ReputationToken.
+      where(track_id: id, type: User::ReputationTokens::CodeContributionToken).
```
Comment (ErikSchierboom) [inline app/models/track.rb:67]: I didn't know exactly what the criterium for a code contributor was. I just went with the `CodeContributionToken`. One could also argue for including the authors and contributors here.

**Comment by ErikSchierboom** [comment]
I think the rubocop failures are addressed in https://github.com/exercism/website/pull/675/files

File: app/models/track.rb (hunk 1 above)
Comment (iHiD) [reply app/models/track.rb:67]: I think this is fine :)

**Comment by iHiD** [comment]
This was great to back to this evening. Thank you 💙

**Pull request merged**